Yafc/Windows/MainScreen.cs

```diff
@@ -163,7 +163,7 @@ public partial class MainScreen : WindowMain, IKeyboardFocus, IProgress<(string,
             else {
                 activePageView = null;
             }
-
+            InputSystem.Instance.SetMouseFocus(null);
```

Comment:
Do we need this? If so, what is it fixing?

Reply:
We do, what that does is handle the case where you've got an open dropdown menu (like, by clicking on an recipe icon and getting the panel where you can change fuels or whatever, or clicking on the module page) and then while that's still open you hit control-tab. The dropdowns close when focus leaves them; normally, that implicitly happens when switching pages because you have to move the mouse up to the tab bar and click it, but if you hit control tab while one is open it would stick around on the new page if we didn't do this.

Will add a comment.